### DISCUSSION TITLE
Installation of the Semantic Web Import Plugin

### DISCUSSION THREAD

**Discussion by lolostar** [gephi/gephi#2664] [Q&A]
Hi all,

I try installing the Semantic Web Import Plugin on Mac OS 12.6 on Gephi (0.9.7 202208031831) and I get the message 👍 

**Certains modules d'extension exigent l'installation du module d'extension org.gephi.io.processor.plugin.
Le module d'extension org.gephi.io.processor.plugin est demandé dans la version 0.9.2.  Le module dextension suivant est concerné :       SemanticWebImport    Note: Missing modules may be published on Update Centers. Please run "Check for Newest" from the "Available Plugins" tab.**

I have no idea how to get "org.gephi.io.processor.plugin." (I suspect something weird in this name) .

Has anybody an idea about that ?

Laurent

**Comment by mbastian** [comment]
Hi Laurent, 

This plugin has not been updated and isn't compatible in Gephi version 0.9.3 and above. The author is @Wimmics. I created a [gephi-plugins issue](https://github.com/gephi/gephi-plugins/issues/280) to track this. It would be great if you could reach out to the author or create an issue on their fork. 

Greetings
